### ISSUE TITLE
how to create a new profile?

### ISSUE THREAD

**Issue by chanduthedev** [quiet/org.quietmodem.Quiet#28]
Hi Brian, I would like to create a new profile with 21000 frequency. Is it possible to create that? if yes, can you please guide me the steps to do that.
Thanks in advance.

**Comment by chanduthedev** [comment]
I tried to change the value of center_frequency to 21000 for **ultrasonic** profile but it didn't worked. Do I need to modify any other values in the profile or any code changes required for this to work?
Thanks in advance.

**Comment by brian-armstrong** [comment]
Yes, it should be as simple as changing the center frequency though you will also need to change interpolation to avoid aliasing at 44.1 kHz sample rate.

Can I ask how you decided on 21000? You're going to make things harder on yourself at that frequency since that will be heavily attenuated by most sound devices.

**Comment by chanduthedev** [comment]
> Yes, it should be as simple as changing the center frequency though you will also need to change interpolation to avoid aliasing at 44.1 kHz sample rate.
> 
> Can I ask how you decided on 21000? You're going to make things harder on yourself at that frequency since that will be heavily attenuated by most sound devices.

**Comment by chanduthedev** [comment]
Hi Brian, thanks for quick reply. Its my bad, bymistake i closed this thread, so reopened it. I am trying to transfer data between mobile devices. and I read ultrasonic frequency is above 20000, so I was trying with 21000. And I also tested with **ultrasonic-experimental**  profile, it works for ~70cm between samsung and apple devices. sometimes it worked for 1.5m as well. But not consistent.
